README.md

```diff
@@ -57,16 +57,43 @@ pywal
 pacman-contrib
 ```
 
-Arch (pacman):
+### Arch
+
+pacman:
 ```bash
 sudo pacman -S pipewire libgtop bluez bluez-utils btop networkmanager dart-sass wl-clipboard brightnessctl swww python gnome-bluetooth-3.0 pacman-contrib
 ```
 
-Arch (AUR):
+AUR:
 ```bash
 yay -S grimblast-git gpu-screen-recorder hyprpicker matugen-bin python-gpustat aylurs-gtk-shell-git
 ```
 
+### Fedora
+
+DNF - with solopasha's [hyprland COPR](https://copr.fedorainfracloud.org/coprs/solopasha/hyprland/)
+```bash
+dnf copr enable solopasha/hyprland
+dnf install pipewire libgtop2 bluez bluez-tools grimblast hyprpicker btop NetworkManager cargo wl-clipboard swww brightnessctl gnome-bluetooth aylurs-gtk-shell
+```
```

Comment:
Optionally remove cargo section and change dnf section to:
> DNF - with solopasha's [hyprland COPR](https://copr.fedorainfracloud.org/coprs/solopasha/hyprland/) and heus-sueh's [matugen COPR](https://copr.fedorainfracloud.org/coprs/heus-sueh/packages/packages/)
```bash
dnf copr enable solopasha/hyprland
dnf install pipewire libgtop2 bluez bluez-tools grimblast hyprpicker btop NetworkManager cargo wl-clipboard swww brightnessctl gnome-bluetooth aylurs-gtk-shell
dnf copr enable heus-sueh/packages
sudo dnf install matugen
```
Subject to testing + @Jas-SinghFSU's preference.